DOC.md

```diff
@@ -154,6 +154,10 @@ Since there would almost always be more jumps if the roots are linked, regular
 `<Tab>` would not work almost all the time, and thus `link_roots` has to stay
 disabled.
 
+If the snippet is not a child, but a root, it can be exited when reaching to
+last node, `$0`, if `exit_roots` is enabled. This setting may avoid unexpected
```

Comment:
I'm not sure we should describe this option at that place, it isn't that strongly related to snippet-insertion as the other 3 that replaced `history`.
I think just shifting this description to the config-section should be fine (maybe with a short recap on snippet-roots, or a hint to snippet-insertion-section on tips for that)

Reply:
I resolved this, should be good now.